README.md

```diff
@@ -1,3 +1,25 @@
-# myDiary
-myDairy is an online journal that allows users to make daily journal entries.
-The site is published at https://sammuel09.github.io/myDiary/
+# Project Title
+myDiary is a web-based journal application. 
+
+## Getting Started
+In order to have a copy of this project, clone this project to your local repository.
```

Comment:
This section should assist a user to have a running copy of this project, cloning the codebase would only copy the project,  users would like to know how to set up and run the project

Reply:
I have updated it now